index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+  
+
+
+  <div>
+    <header class="nav">
+    <a href="" class="m-logo" >My Logo</a>
+    <nav>
+    <ul>
+         
+         <li class="link-right">
+              
+              <a href="/someurl" class="inline"><b><img src="images/menu.PNG" alt="ss"></b></a>
+          </li>
+        
+         </ul>
+         </nav>
+     </header>
+
+
+
+        <section class="section1">
+          
+          <img src="images/l1.PNG" alt="abc" class="l1">
+          <img src="images/m.PNG" alt="abc" class="m">
+          <img src="images/o.PNG" alt="abc" class="m">
+          <img src="images/r.png"  alt="abc" class="image3 dkl" >
+
+        
+
+  
+        </section>
+
+
+
+
+
+
+
+
+        <div class="section2">
+          
+          <h1 class="h2 imelex">I’m Aziz un Nabi </h1>
+                        <h1 class="h2 imelex gl">Glad to see you!</h1>
+                        <p  class="sec3p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                        <h5 class="ets" >ETS CONNECT</h5>
+                          
+                           <div class="sec2image" >
+                            <ul class="social-menue">
+                      <li><img src="images/twitter.PNG" alt="abc"></li>
+                      <li><img src="images/linkedinicon.PNG" alt="abc"></li>
+                      <li> <img src="images/medium.PNG" alt="abc"></li>
+                      <li> <img src="images/github.PNG" alt="abc"></li>
+                      <li><img src="images/angellist.PNG" alt="abc"></li>
+                     </ul>
+                   </div>
+
```

Comment:
- [ ] According to the project requirements. The container of the section should be a `section` tag and not a `div` tag. Kindly move all this code inside the `<section class="section1">` container 👨🏻‍🎨 
* It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.